bundles/org.openhab.binding.panamaxfurman/src/main/java/org/openhab/binding/panamaxfurman/internal/PanamaxFurmanTelnetConfiguration.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.panamaxfurman.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Contains fields mapping thing configuration parameters.
+ *
+ * @author Dave Badia - Initial contribution
+ */
+@NonNullByDefault
+public class PanamaxFurmanTelnetConfiguration {
+
+    /**
+     * The hostname (or IP Address) of the Panamax/Furman device
+     */
+    private String address = "";
+
+    /**
+     * The telnet port
+     */
+    private int telnetPort = 23;
+
+    /**
+     * Socket read timeout in seconds
+     */
+    private int socketReadTimeoutSeconds = 23;
```

Comment:
This is no longer used, as it is set to 3 times the pollinginterval in file: PanamaxFurmanTelnetHandler.java:47
Also remove the set/get method

Reply:
Good catch